api/controllers/game/move.js

```diff
@@ -36,19 +38,20 @@ module.exports = async function (req, res) {
     return res.ok();
   } catch (err) {
     // ensure the game is unlocked
-    if (game?.lock) {
-      try {
-        await sails.helpers.unlockGame(game.lock);
-      } catch (err) {
-        // fall through for generic error handling
-      }
+    try {
+      await sails.helpers.unlockGame(game.lock);
+    } catch (err) {
+      // fall through for generic error handling
     }
 
+    const message = err?.message ?? err ?? 'Error making move';
     switch (err?.code) {
-      case 'FORBIDDEN':
-        return res.forbidden({ message: err.message });
+      case CustomErrorType.FORBIDDEN:
+        return res.forbidden({ message });
+      case CustomErrorType.BAD_REQUEST:
+        return res.badRequest({ message });
       default:
-        return res.badRequest({ message: err.message });
+        return res.serverError({ message });
```

Comment:
Now the catch-all for unexpected errors e.g. failure to write to db is 500, and the specific errors associated w/ 403 and 400 will respond appropriately